src/improvements/template/UnPauseSuperchainConfigAfterU16.sol

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import {VmSafe} from "forge-std/Vm.sol";
+import {stdToml} from "forge-std/StdToml.sol";
+
+import {L2TaskBase} from "src/improvements/tasks/types/L2TaskBase.sol";
+import {SuperchainAddressRegistry} from "src/improvements/SuperchainAddressRegistry.sol";
+
+import {IOptimismPortal2} from "lib/optimism/packages/contracts-bedrock/interfaces/safe/IDeputyGuardianModule.sol";
+
+import {Action} from "src/libraries/MultisigTypes.sol";
+
+interface IETHLockbox {
+    // function systemConfig() external view returns (ISystemConfig);
+
+    function paused() external view returns (bool);
+}
+
+interface ISuperchainConfig {
+    function guardian() external view returns (address);
+
+    function pause(address _identifier) external;
+
+    function unpause(address _identifier) external;
+
+    function paused() external view returns (bool);
+
+    function paused(address _identifier) external view returns (bool);
+}
+
+/// @title UnPauseSuperchainConfigTemplateAfterU16 template.
+contract UnPauseSuperchainConfigTemplateAfterU16 is L2TaskBase {
+    using stdToml for string;
+
+    /// @notice Identifier of the eth_lockbox used in the SuperchainConfig loaded from TOML
+    address public identifier;
+
+    /// @notice SuperchainConfig SC contract instance
+    ISuperchainConfig public sc;
+
+    /// @notice Returns the string identifier for the safe executing this transaction.
+    function safeAddressString() public pure override returns (string memory) {
+        return "SecurityCouncil";
+    }
+
+    /// @notice Returns string identifiers for addresses that are expected to have their storage written to.
+    function _taskStorageWrites() internal pure override returns (string[] memory) {
+        string[] memory storageWrites = new string[](2);
+        storageWrites[0] = "SuperchainConfig";
+        storageWrites[1] = safeAddressString();
```

Comment:
Is this because of the LivenessGuard?

Reply:
Yeah, but I realized that we don't need to put the safe somehow.

Reply:
yeah fixed here: 259b39c8a613524a4fa7c5e4d7230e468c06f2db

Reply:
ok, I'd imagine if you're using the security council mainnet address in this task you'll need the Livenessguard in there like this: https://github.com/ethereum-optimism/superchain-ops/blob/main/src/improvements/template/GnosisSafeRemoveOwner.sol#L45

Reply:
You think I should do add still the SC into? 
For me that make sense but somehow here it seems to be by default?

Reply:
Not sure I'm following what you mean. I'm saying, if this is simulated with the mainnet security council address, you'll likely get an error. Your test task is in the sepolia directory right now.

Reply:
Oh that's precisely my point! 
this is the liveness module on sep for the SC SEP-> https://sepolia.etherscan.io/address/0xEB3eF34ACF1a6C1630807495bCC07ED3e7B0177e#code
this is the livenessguard on  sep  -> https://sepolia.etherscan.io/address/0xc26977310bC89DAee5823C2e2a73195E85382cC7#readContract

I wonder how this is possible that the storage access is not reverting since I didn't add the the livenessModule logic into the storage access. 
This should even revert on sepolia no?

Reply:
Ah, never mind, the simualtion executes on the Guardian not the SecurityCouncil. So you won't see the LivenessGuard state changes locally, only in Tenderly.